### DISCUSSION TITLE
How to get historical options chain data?

### DISCUSSION THREAD

**Discussion by ted-love** [ranaroussi/yfinance#1842] [Q&A]
I am trying to get historical options chain data with varying strikes, specifically on the date 2012/06/21. It doesn't have to be 5m ticks, closing price would be fine. I have tried:
```
opt = yf.Ticker('AMZN120629P00010000').history('max')
```
But I get the error
```
Exception: _fetch_ticker_tz() has returned None for 'AMZN120629P00020000', will cause problems. Investigate
```
And if I try:
```
opt =  yf.Ticker('AMZN120629P00010000')
opt.info
AttributeError: module 'requests.cookies' has no attribute 'clear'
```
Amazon was trading around $10 then so there should have been an option with that date and expiry. I even tried an option closer today like 'AMZN240110C00160000', but still get the same error. Any ideas would be greatly appreciated.

**Comment by ms82494** [comment]
I think historical prices are only available for current (non-expired) options contracts.

**Reply by mktcowboy** [reply]
is options data still available? i had a bunch of scripts pulling data now none of them work, getting the error "Available expiration are: []"

**Comment by jh4119** [comment]
@ted-love were you able to find a solution to this? I am also looking for historical EOD option data.
